### ISSUE TITLE
Cleanup StrykerOptions in Unit tests

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/stryker-net#259]
Right now a lot of Unit Tests create instances of the StrykerOptions class. The constructor doesn't have any default values so each Unit Test has to pass all parameters, even though most unit tests only need the default values.

I would like to see a solution where in the Unit Tests an instance of the StrykerOptions can be created by only passing values that need to be "non default". For example:

``` csharp
StrykerOptions options = new StrykerOptions("", "Console", "", 1000, null, "trace", false, 1, 90, 80, 70);
```
Could become:
``` csharp
StrykerOptions options = new StrykerOptions(timeoutMS: 1000);
```
This would make maintenance of the Unit Tests a lot better. Also it improves readability.

**Comment by richardwerkman** [comment]
Cool! If you have any questions let us know 👍

**Comment by Mobrockers** [comment]
Closed by #274
